R/cookie-parser.R

```diff
@@ -25,5 +25,5 @@ parseCookies <- function(cookie){
   cookies <- lapply(cookieList, "[[", 2)
   names(cookies) <- sapply(cookieList, "[[", 1)
 
-  return(lapply(cookies, URLdecode))
+  return(lapply(cookies, decodeURI))
```

Comment:
As mentioned below, `decodeURI` is vectorized, so it's faster to not use `lapply` (though you may need `as.list` if it's important that the return value is a list instead of character vector).